### PR TITLE
Gidk 2x8wrw new gst stage

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/gst/gst.component.html [hunk 1]
```diff
@@ -132,23 +132,24 @@
...
+                                            <ng-container *ngIf="isMonthSelected">
+                                                ({{ moment(selectedMonth).format('MMMM YYYY') }})
+                                            </ng-container>
+                                            <ng-container *ngIf="!isMonthSelected">
+                                                ({{moment(currentPeriod?.from, giddhDateFormat).format('DD MMM YYYY')}}
+                                                - {{moment(currentPeriod?.to, giddhDateFormat).format('DD MMM YYYY')}})
+                                            </ng-container>
+
+                                        </b> &nbsp;&nbsp;
+                                        <!-- <span class="label text-uppercase return-filed "> Filed </span> -->
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/gst.component.html:149]: Remove commented code

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/2x8wrw 
Developer - @shalinee-walkover @ravinderwalkover 
Description - cherrypick gstr reponsive code

File: apps/web-giddh/src/app/gst/gstR3/gstR3.component.html [hunk 2]
```diff
@@ -141,729 +144,744 @@
...
+                                        <td class="text-right "><i class="icon-rupees font-11"></i>
+                                            <span class="d-inline-flex">
+                                                <amount-field [amount]="gstr3BData?.sup_details?.osup_det?.iamt"
+                                                    [currencySymbol]="false" [currencyCode]="false">
+                                                </amount-field>
+                                            </span>
+                                        </td>
+                                        <td class="text-right "><i class="icon-rupees font-11"></i>
                                             <span class="d-inline-flex">
-                                                <amount-field [amount]="item?.pos" [currencySymbol]="false"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/gstR3/gstR3.component.html:389]: Please check bindings are getting changed

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/gst/filing/filing.component.ts [hunk 3]
```diff
@@ -91,7 +91,7 @@ export class FilingComponent implements OnInit, OnDestroy {
     public ngOnInit() {
         document.querySelector('body').classList.add('gst-sidebar-open');
         this.breakpointObserver
-        .observe(['(max-width: 767px)'])
+        .observe(['(max-width: 768px)'])
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/gst/filing/filing.component.ts:94]: @shalinee-walkover why we are using 768px in place of 767px? ideal media query point is 767px.

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/filing/filing.component.ts:94]: For the ipad view , 768px is ideal resolution.

File: apps/web-giddh/src/app/gst/gst.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gst.component.html:149]: REmoved

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 4]
```diff
@@ -230,7 +232,15 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
     public get shouldShowBackButton(): boolean {
         return this.router.url && (ROUTES_WITH_HEADER_BACK_BUTTON.includes(this.router.url));
     }
-
+    /**
+    * Opens the GST side menu in responsive mode
+    *
+    * @memberof HeaderComponent
+    */
+    public openGstSideMenu(): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:240]: Please move this function to the bottom

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 4 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/shared/header/header.component.ts:240]: Please check this

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**